.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - '*'
+
```

Comment:
Shouldn't this run only on the master branch?

Reply:
It will only trigger when creating tags, which I don't think we do unless we release a new version.

You can't have a filter on where the tag comes form sadly, but maybe we can limit it to a a format of tags, like "v1.0.0" with "v*"

Reply:
I don't understand...
You should only deploy when running on master because that's the released version. If you check rubygems now you see all your work-in-progress packages release to production (logstash pulls the latest one), which isn't good IMO.